.ci/sonarAnalysis.stages

```diff
@@ -0,0 +1,19 @@
+def call(def propertiesFolderPath) {
+    stage('Sonar Analysis') {
+        def gitURL = env.ghprbAuthorRepoGitUrl ?: env.GIT_URL
+        def project = util.getProjectGroupName(util.getProject(gitURL))[1]
+        if(["optaplanner", "drools", "appformer", "jbpm", "drools-wb", "kie-soup", "droolsjbpm-integration", "kie-wb-common", "openshift-drools-hacep", "optaweb-employee-rostering"].contains(project))
```

Comment:
is the list of repositories complete?
I miss for instance optaweb-vehicle-routing there

Reply:
@mareknovotny  they are the projects to execute sonar analysis. The list is taken from here https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/3f5746abbc7380c97878982d91cb2b2ccdc5a303/.ci/pullrequest.stages#L14
Probably @mbiarnes can clarify this better than me.

Reply:
@yurloc the sonarcloud analysis in optaweb-vehicle-routing is handled by travis. Do you prefer keeping it like that?

Reply:
@mareknovotny @mbiarnes ^

Reply:
I'm not against unification and I don't care whether SonarCloud analysis runs on Jenkins or Travis. What I do care about is a fast response time. So if I update a PR I want to get the analysis results within 20 minutes.

Also, the SonarCloud PR check stopped working on OptaWeb ER a few months ago. So until that is fixed I don't want to take the same path with OptaWeb VR.

Reply:
@yurloc @rsynek my opinion is to run the sonarcloud analyis by this change, so add optaweb-vehicle-routing to this list. It is more consistent I would say.

Reply:
I thkn you can add `optaweb-vehicle-routing` on the list if that doesn't break anything. Let me know what I need change in `optaweb-vehicle-routing/pom.xml` in order to make the analysis actually run when building on Jenkins.

Reply:
`optaweb-vehicle-routing` added. ready to merge